src/main/java/com/pollapp/controller/PollController.java

```diff
@@ -86,6 +95,18 @@ public Poll addCategoryToPoll(@PathVariable long pollId, @PathVariable int categ
         poll.getCategories().add(categoryRepository.findOne(categoryId));
         return pollRepository.save(poll);
     }
+    
+    @PostMapping("/{pollId}/closed/{days}/{hours}")
+    public Poll addHoursToPoll(@PathVariable long pollId, @PathVariable int days, @PathVariable int hours) {
```

Comment:
Now if i think of this, created and closed attributes are part of poll entity so while we create poll from form those attributes should be properly set in created poll and then simply send this created poll to backend where we save it. Or create poll, save it and then update it with values. Because i think we don't need this additional endpoint while we can do this logic in endpoints like post /polls or put /polls which we already have.

Reply:
I think it's not just the issue of ongoing/closed, as you still need to include option of setting timeframes from poll closing.